### PR TITLE
Updated VOC documentation

### PR DESCRIPTION
**Pull request by molobrakos**
**Description:**
The Volvo on call device tracker was transformed into its own component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/volvooncall.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ha_category: Hub
+ha_release: 0.39
+---
+
+
+The `volvooncall` platform offers integrates with the [Volvo On Call](http://www.volvocars.com/intl/own/connectivity/volvo-on-call) cloud service and offers presence detection as well as sensors such as odometer and fuel level.
+
+To use Volvo On Call in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
```
Comment (Landrash) [inline source/_components/volvooncall.markdown]: Looks good but I suggest you have a minimal example and this one as a advanced example.

Comment (molobrakos) [inline source/_components/volvooncall.markdown]: Good point, I'll fix. Am I supposed to add sensor.volvooncall, switch.volvooncall etc as well? (currently they can only instantiated indirectly by the component, not by the user)

Comment (molobrakos) [inline source/_components/volvooncall.markdown]: Updated.

**Comment by tomoqv** [comment]
Hello!
could somebody explain to me what I am missing in my VOC config? I am not getting any device_tracker info for my Volvo. Pls see this thread: https://community.home-assistant.io/t/trying-to-add-volvo-on-call-position/17459/6

Thank you!
Tomas

**Comment by Landrash** [comment]
@tomoqv Please keep support to the forums or gitter chat.

**Pull request merged**